HybridLocalTemplate/install.js

```diff
@@ -10,6 +10,13 @@ for (var sdkDependency in packageJson.sdkDependencies) {
     var repoUrlWithBranch = packageJson.sdkDependencies[sdkDependency];
     var parts = repoUrlWithBranch.split('#'), repoUrl = parts[0], branch = parts.length > 1 ? parts[1] : 'master';
     var targetDir = path.join('mobile_sdk', sdkDependency);
+
+    if (sdkDependency == 'SalesforceMobileSDK-Android' && branch.match(/v\d+\.\d+\.\d+/)) {
+        console.log('Skipping dependency installation of production SalesforceMobileSDK-Android release.  It is only used for source builds in pre-production.  Production builds default to release repository artifacts.');
+        continue;
+    }
```

Comment:
They might have a left over mobile sdk, maybe we should warn them like we do on line 20, or maybe move the line 20 check earlier.

Reply:
We should do the same in the other hybrid templates: [hybrid remote](https://github.com/forcedotcom/SalesforceMobileSDK-Templates/blob/dev/HybridRemoteTemplate/install.js) and [hybrid lwc](https://github.com/forcedotcom/SalesforceMobileSDK-Templates/blob/dev/HybridLwcTemplate/install.js).

Where you planning to deal with native templates in this PR or another PR?

Reply:
Before this wraps up, each of the related pull requests will need some finalization, including updating the remaining hybrid templates, at least, and switching to formal branches.

Reply:
👍